modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/constants/alertDownloadType.js

```diff
@@ -9,7 +9,9 @@
  * distribution rights of the Software.
  */
 
+import i18n from '../../../../common/I18n';
+
 export const ALERT_DOWNLOAD_TYPE = {
-	danger: 'danger',
-	success: 'success',
+	danger: i18n.translate('danger'),
```

Comment:
Aqui tem certeza que altera os alerts? Porque essa constante é pro tipo de alert. Success pra mensagens de sucesso e danger pra mensagens de erro. 
Ex: `[ALERT_DOWNLOAD_TYPE.danger]: 'Unable to download key, please try again.',`

Reply:
Corrigido! O texto "Error" e "Success" do alert vem by default do Clay. Pra modificá-lo, adicionei a prop title e, assim, a chave de tradução. Fiz os testes e funcionou corretamente .